configurations/manager/debian10.yaml

```diff
@@ -1,4 +1,5 @@
 ami_id_monitor: 'ami-074f8bbb689b1c1a0'  # Debian buster image - debian-10-amd64-20230601-1398
 ami_monitor_user: 'admin'
+monitor_branch: 'branch-4.6'  # Pass it as debian image is not the formal monitor image
```

Comment:
at as a temporary hack, this would work.

but I think no one would remember updating this when updating the images or the default configuration.

I don't have a better idea now, but using different machines for monitor and manager, so those won't be conflicting all the time.